Sources/NIOConcurrencyHelpers/atomics.swift

```diff
@@ -19,11 +19,17 @@ import Darwin
 fileprivate func sys_sched_yield() {
     pthread_yield_np()
 }
-#else
+#elseif os(Android) || os(Linux)
 import Glibc
 fileprivate func sys_sched_yield() {
     _ = sched_yield()
 }
+#elseif os(Windows)
+import ucrt
+import WinSDK
+fileprivate func sys_sched_yield() {
```

Comment:
this is only used in deprecated functionality, so if you want, just do nothing on Windows